pkg/pillar/cmd/monitor/ipc_server.go

```diff
@@ -169,14 +170,24 @@ func (s *monitorIPCServer) sendIpcMessage(t string, msg any) error {
 	defer s.Unlock()
 
 	var err error
+	var data []byte
 
-	if data, err := json.Marshal(msg); err == nil {
+	if data, err = json.Marshal(msg); err == nil {
 		ipcMessage := ipcMessage{Type: t, Message: json.RawMessage(data)}
 		if data, err = json.Marshal(ipcMessage); err == nil {
-			log.Noticef("Sending IPC message: %s", string(data))
+			if t == "TpmLogs" {
+				log.Noticef("Sending IPC message: %s", t)
```

Comment:
what about use `Sending IPC TPM log message: %s` here so it becomes easier to find these in the logs?

Reply:
@rene I prefer to have a common prefix to grep the log. it is easier to filter later

Reply:
ok